serverless-functions/src/functions/common/flex/taskrouter/update-worker-attributes.js

```diff
@@ -26,7 +26,7 @@ exports.handler = prepareFlexFunction(requiredParameters, async (context, event,
       workerSid,
       attributesUpdate,
     });
-    const { status, worker } = result;
+    const { status, data: worker } = result;
```

Comment:
This was answered in a separate Slack conversation with Sam. Only when an operation requires complex logic- such as a GET to retrieve attributes, then manipulating the response with new attributes, then a POST to update attributes- does it make sense to put that in a wrapper function. Otherwise the Twilio helper library request will be executed directly in the function that needs to make the request.